wolfcrypt/src/integer.c

```diff
@@ -421,6 +421,10 @@ int mp_grow (mp_int * a, int size)
   int     i;
   mp_digit *tmp;
 
+#ifdef WOLFSSL_MP_PREALLOC
```

Comment:
This puts the restriction on all mp_int's even if not involved with WOLFSSL_BIGNUM. Can this restriction be put at a spot where it only effects WOLFSSL_BIGNUM structures?

Reply:
preAlloc flag in mp_int to manage it dynamically, now.